packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -251,13 +296,27 @@ export const FieldContainer = (inProps: PropsWithChildren<FieldContainerProps>)
                 )}
                 <InputContainer ownerState={ownerState} {...slotProps?.inputContainer}>
                     {children}
-                    {hasError && (
-                        <Error error {...slotProps?.error}>
-                            {error}
-                        </Error>
-                    )}
-                    {hasWarning && !hasError && <Warning {...slotProps?.warning}>{warning}</Warning>}
-                    {helperText && !hasError && !hasWarning && <HelperText {...slotProps?.helperText}>{helperText}</HelperText>}
+                    <HelperTextWrapper {...slotProps?.helperTextWrapper}>
+                        {hasError && (
+                            <Error error {...slotProps?.error}>
+                                {helperTextIcon && helperTextIcon}
+                                <HelperTextContainer>{error}</HelperTextContainer>
```

Comment:
Please apply the slots props to `HelperTextContainer`.
(Same for the items below)

```suggestion
                                <HelperTextContainer {...slotProps?.helperTextContainer}>{error}</HelperTextContainer>
```